Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Build this Docker container from `protocol` directory with:
 #   docker build -t <username>/<imagename> .
 # Execute the voting system with:
-#   docker run <username>/<imagename> --network=<network>
+#   docker run --volume=<path-to-config-dir>:/protocol/core/docker-config:ro <username>/<imagename> --network=<network>
```

Comment:
If a user were to try to run this on GCS, would they need to pre-load the image with a config file?

Reply:
Yeah, the config file would ultimately have to be loaded onto the same instance that `docker run` gets called on.

Reply:
Is that possible to do without ssh-ing in after the instance starts?